Command/CreateTweetContentCommand.php

```diff
@@ -25,9 +25,10 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $repository = $this->getContainer()->get('ezpublish.api.repository');
-        $repository->setCurrentUser(
-            $repository->getUserService()->loadUserByLogin('admin')
-        );
+        $userService = $repository->getUserService();
```

Comment:
Not related to this line, but there's a typo in this change's commit message: "Remove call to *depracated* method setCurrentUser".